src/App.js

```diff
@@ -3,13 +3,30 @@ import styles from "./App.module.css";
 import { Text } from "./components/common";
 import StoreProvider from "./store";
 import Counter from "./components/Counter";
+import { HeadingL } from "./components/HeadingL/HeadingL.jsx"
+import { HeadingXL } from './components/HeadingXL/HeadingXL.jsx'
 
 
 function App() {
```

Comment:
Let's revert the `App.js` component to not create conflicts.

Reply:
Done